ChangeLog

```diff
@@ -1,52 +1,65 @@
-07/09/2016 - compatibility with Coq 8.7 and several small fixes -
-	version 1.6.2 and upcomming version 1.7
-	* Compatibility with Coq 8.7
+23/04/2018 - compatibility with Coq 8.7 and several small fixes -
+	upcoming version 1.7
```

Comment:
remove "upcoming"?

Reply:
My intention was to update this line at the last minute, including the correct date of the release.